include/lazy_importer.hpp

```diff
@@ -381,9 +379,9 @@ namespace li { namespace detail {
     }
 
     struct exports_directory {
+        unsigned long                      _ied_size;
```

Comment:
doesn't make a difference since it'll be inlined, but strange for compiler to complain about padding at the end of structure but be ok with it in middle of it.

Reply:
I see. This warning may be thrown because of another flag I didn't see, there is a bunch I need to check.